salt/states/boto_secgroup.py

```diff
@@ -577,16 +577,19 @@ def absent(
     '''
     ret = {'name': name, 'result': True, 'comment': '', 'changes': {}}
 
-    sg = __salt__['boto_secgroup.get_config'](name, True, region, key, keyid,
-                                              profile, vpc_id, vpc_name)
+    sg = __salt__['boto_secgroup.get_config'](name=name, group_id=None, region=region, key=key,
```

Comment:
Oh, I think you're right. After looking at the `get_config` code in the module, that shouldn't be a boolean at all, and since `group_id` isn't available in this state, I think `None` is the correct setting.